app/opportunities/OpportunityList.tsx

```diff
@@ -169,95 +186,85 @@ export default function OpportunityCardsPage() {
-              <SheetTrigger asChild>
-                <Button variant="outline" size="sm" className="px-3">
-                  <Filter className="h-4 w-4" />
-                </Button>
-              </SheetTrigger>
-              <SheetContent side="bottom" className="h-[400px]">
-                <SheetHeader>
-                  <SheetTitle>Filter & Sort</SheetTitle>
-                  <SheetDescription>
-                    Customize your opportunity search
-                  </SheetDescription>
-                </SheetHeader>
-                <div className="mt-4 space-y-4 px-4">
-                  <div>
-                    <label className="mb-2 block text-sm font-medium">
-                      Filter by Type
-                    </label>
-                    <Select value={filterType} onValueChange={setFilterType}>
-                      <SelectTrigger>
-                        <SelectValue placeholder="All Types" />
-                      </SelectTrigger>
-                      <SelectContent>
-                        <SelectItem value="all">All Types</SelectItem>
-                        <SelectItem value="hackathon">Hackathons</SelectItem>
-                        <SelectItem value="grant">Grants</SelectItem>
-                        <SelectItem value="competition">
-                          Competitions
-                        </SelectItem>
-                        <SelectItem value="ideathon">Ideathons</SelectItem>
-                        <SelectItem value="others">Others</SelectItem>
-                      </SelectContent>
-                    </Select>
-                  </div>
 
-                  <div>
-                    <label className="mb-2 block text-sm font-medium">
-                      Sort by
-                    </label>
-                    <Select value={sortBy} onValueChange={setSortBy}>
-                      <SelectTrigger>
-                        <SelectValue placeholder="Newest First" />
-                      </SelectTrigger>
-                      <SelectContent>
-                        <SelectItem value="newest">Newest First</SelectItem>
-                        <SelectItem value="oldest">Oldest First</SelectItem>
-                        <SelectItem value="title">Title A-Z</SelectItem>
-                        <SelectItem value="startDate">Start Date</SelectItem>
-                      </SelectContent>
-                    </Select>
-                  </div>
+        {/* Mobile: Search */}
+        <div className="mb-5 lg:hidden">
+          {/* Tag Badges - Above Search Bar */}
+          <div className="mb-3 flex flex-wrap items-center gap-2">
+            {AVAILABLE_TAGS.map((tag) => {
+              const isSelected = selectedTags.includes(tag);
+              return (
+                <Badge
+                  key={tag}
+                  variant={isSelected ? "default" : "outline"}
+                  className={`cursor-pointer px-3 py-1 text-sm font-semibold ${
```

Comment:
can we remove `font-semibold` from this? Also can color be same as whatever is tags color in opportunity right now?

Reply:
resolved by me